addons/website/static/src/snippets/s_color_blocks_2/000.scss

```diff
@@ -1,12 +1,9 @@
 .s_color_blocks_2 {
-    .row {
-        display: flex;
-        flex-flow: row wrap;
-
-        // Fix for safari browser as it 'supports' flex but not with the right
-        // behavior
-        &::before, &::after {
-            width: 0;
+    &.o_full_screen_height,
+    &.o_half_screen_height {
+        > * {
+            display: flex;
+            flex-grow: 1;
```

Comment:
@qsm-odoo it uses this:

```scss
.o_full_screen_height {
    display: flex;
    flex-direction: column;
    justify-content: space-around;
    min-height: 100vh;
}
```

To use space-around vertically in the sections. This centers section content when there is only one element and would give some space to the scroll down button if there was any extra space. It's how it was initially done in blog, so I saw no reason to change it. If the button is now absolutely positioned, this can be changed, but I feel like using display flex for vertical centering is still the best option, be it with justify-content: space-around or center.

Reply:
Meh. Our options are kinda conflicting.

Yes, the scroll button will now be absolutely position as it makes more sense in some snippets and yields the same result in the other ones. So here, we won't need direction: column anymore, there will only be one staticly-positioned elements in snippet section and it will be the container div. However: @Cocographique need to check that still working in blogs.

However, unsure about the rest: the half/full screen option forces vertical centering... but after that some snippets (like 3 columns) propose vertical alignement on the row... And in the case of the color block here, we don't want vertical centering but full height of the row and col. @Cocographique @sbuhl Is this the only snippet where we have that problem? Otherwise maybe it should not be specific scss for the snippet but a class to add on the snippets which need to work differently with the half/full height options. Or maybe we should in fact review the vertical alignment option to:
- Be visible on all snippets when the half/full height option is checked.
- Apply the alignment on the columns when there are some or on the container when there is no column.

Last suggestion is probably the best but not sure it is as easy to implement.

Reply:
@sbuhl Maybe we should speak about this with @Cocographique, can you come this afternoon maybe ? :)

Reply:
This is the case for Masonry also, forgot to point this before, sorry. 
And check with Images Wall ( https://github.com/odoo/odoo/pull/45096 )

Reply:
I'll come down from my ivory tower ;)

Reply:
@sbuhl What did we decide about this finally? In this task, not in this task? Final specs? I am lost :/

Reply:
@qsm-odoo Not here as we don't have final specs yet.

Reply:
@Cocographique Can I ask you to remove that commit from this PR then? Sorry.